go/cmd/zkctl/zkctl.go

```diff
@@ -61,18 +60,10 @@ func main() {
 	defer logutil.Flush()
 
 	fs := pflag.NewFlagSet("zkctl", pflag.ExitOnError)
-	fs.Usage = func() {
-		fmt.Fprintf(os.Stderr, usage)
-	}
-
 	log.RegisterFlags(fs)
 	logutil.RegisterFlags(fs)
-	args := servenv.ParseFlagsWithArgs("zkctl")
+	servenv.ParseFlagsWithArgs("zkctl")
```

Comment:
why are we parsing flags with args but then not doing anything with them?

Reply:
ah, i see, [this](https://github.com/vitessio/vitess/pull/11331/files/8a6f06a044fc7de1d36042bd991b0ace8bb92548..f20d7a532e15e3ba0b21663f7e4a14db1b0320cf#diff-832edc3db69da386548b153cd7268a79e0655592b1061dfb3cbadc1d03977a35R70) should be `args[0]`

Reply:
args[0] and pflag.Arg(0) is producing same result so I used later one.. If you feel strongly then I can use args[0] .. I would advice if this is the only outstanding issue then please approve it and i will do that change before I update and merge ...

Reply:
you should use `args` because it's _technically_ different (there's some extra stuff happening under the hood to facilitate the migration in v14)